workers/unity/Assets/Playground/Config/EntityTemplates.cs

```diff
@@ -77,8 +77,12 @@ public static EntityTemplate CreateCubeEntityTemplate(Vector3 location)
 
             TransformSynchronizationHelper.AddTransformSynchronizationComponents(template, WorkerUtils.UnityGameLogic, Quaternion.identity, location);
 
+            var query = InterestQuery.Query(Constraint.RelativeCylinder(radius: CheckoutRadius));
```

Comment:
These now imply interest in everything, including ACL and Interest components.

Reply:
ugh forgot that